app/api/access_codes.py

```diff
@@ -36,7 +36,8 @@ def before_post(self, args, kwargs, data):
         """
         require_relationship(['event', 'user'], data)
         if not has_access('is_coorganizer', event_id=data['event']):
-            raise ForbiddenError({'source': ''}, "Minimum Organizer access required")
+            raise ForbiddenError({'parameter': 'event_id'},
```

Comment:
There is no parameter `event_id`, change this to `pointer: /data/event`

Reply:
ok